templates/postgres-sts.yaml

```diff
@@ -56,42 +56,16 @@ spec:
       containers:
       - name: postgres
         env:
-        - name: POSTGRES_DB
-          value: testdb
-        - name: POSTGRES_USER
-          value: postgres
-        - name: POSTGRES_PASSWORD
+        - name: POSTGRESQL_ADMIN_PASSWORD
           valueFrom:
             secretKeyRef:
               key: rootpassword
               name: {{.SecretName}}
         image: {{.Base.Spec.Postgres.Image}}:{{.Base.Spec.Postgres.Version}}
-        imagePullPolicy: IfNotPresent
-        livenessProbe:
-          exec:
-            command:
-            - bash
-            - -c
-            - psql -w -U $POSTGRES_USER -d $POSTGRES_DB -c SELECT 1
-          failureThreshold: 3
-          initialDelaySeconds: 30
-          periodSeconds: 20
-          successThreshold: 1
-          timeoutSeconds: 5
+        imagePullPolicy: Always
         ports:
         - containerPort: 5432
           name: postgres
-        readinessProbe:
```

Comment:
So we do not have liveness and readiness probes now?

Reply:
The readinessProbe used the POSTGRES_USER and POSTGRES_DB variables, and we removed those

Reply:
We will need to add them back - issue here https://github.com/opendatahub-io/airflow-on-k8s-operator/issues/2